userspace/libsinsp/sinsp.h

```diff
@@ -193,6 +193,40 @@ class sinsp_proc_metainfo
 	ppm_proclist_info* m_pli;
 };
 
+/*!
+  \brief Scap possible modes
```

Comment:
```suggestion
  \brief Sinsp possible modes
```

Reply:
Thanks! Applied and folded into the relevant commit